spec/03-plugins/23-rate-limiting/05-integration_spec.lua

```diff
@@ -98,11 +75,11 @@ describe("Plugin: rate-limiting (integration)", function()
         -- https://github.com/Kong/kong/issues/3292
 
         lazy_setup(function()
-          flush_redis(red, REDIS_DB_1)
```

Comment:
This operation used `red:flushall()` underneath which clears all databases so it's not necessary to run it for different dbs.